webpack.main.config.js

```diff
@@ -2,11 +2,13 @@ const path = require('path');
 const webpack = require('webpack');
 const merge = require('webpack-merge');
 const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const NodemonPlugin = require('nodemon-webpack-plugin');
 
 const baseConfig = require('./webpack.base.config');
 
 module.exports = merge.smart(baseConfig, {
     target: 'electron-main',
+    watch: true,
```

Comment:
If you add this, I believe you need the inverse on `webpack.main.prod.config.js` since it inherits the config from this.